README.md

```diff
@@ -1,2 +1,62 @@
 # deploy-llm-bot
-Configuration-as-Code repo to deploy LLM Assist Bot
+Configuration-as-Code repo to deploy LLM Assist Bot.
+
+## Accessing the LLM VMs
+
+A combination of security measures is employed to secure access to the LLM Virtual Machines (VMs):
+
+- [Hashicorp Vault](https://www.hashicorp.com/products/vault): Signing SSH certificate at runtime, ensuring only our team members have access to the server. This certificate will expire after 8h.
+- [Github Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens): Validating that only members of the dev team can SSH into the VMs.
+- [Cloudflare Tunnel](https://github.com/cloudflare/cloudflared): The VMs do not have public IP addresses exposing to the internet, enhancing their security.
+
+As such, following the steps below to ensure you can access to the LLM VMs through a combination of secure tunnels, authenticated access, and signed certificates:
+
+### Install `cloudflared`
+
+Before proceeding, you must install `cloudflared` on your local machine. This tool creates a secure tunnel to the VMs, facilitating a secure connection without a public IP. For installation instructions, visit the [Cloudflare Tunnel documentation](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/downloads/).
```

Comment:
Add a TL;DR

Reply:
The linked Cloudflare instruction is already very short. I don't think we can do a TL;DR as installation is depending on the OS you are operating on.

Reply:
Fair enough.